docker-sync.yml

```diff
@@ -24,13 +24,13 @@ syncs:
 #    sync_excludes: ['.DS_Store', '*.pyc', '*.tmp', '.git', '.idea']
 #    watch_excludes: ['.*\.DS_Store', '.*\.pyc', '.*\.tmp', '.*/\.git', '.*/\.idea']
 
-#  emberosf-sync:
```

Comment:
Those docker change doesn't seem to be related to the purpose of this pr. Are those necessary?